### PR TITLE
Added a section for deployment on Azure

### PR DESCRIPTION
**Pull request by MaanavD**
Co-authored-by: rem20806 t-remcfa@microsoft.com
Co-authored-by: mohona17 t-moahme@microsoft.com

Not a robust and complete deployment guide, but more than enough to get Kubeflow deployed and running on Azure. 

Upcoming PRs will have information on authentication. Will also serve as a base to be expanded upon during the upcoming doc sprint.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/780)

### CODE REVIEW

**Comment by abhi-g** [comment]
/ok-to-test
@MaanavD your commit might have a different identity than the github id you used to create the PR. Please see the CLA message

**Comment by MaanavD** [comment]
@abhi-g I checked my CLA, and all the e-mails I use at all have been signed. Is there a chance that it's checking my co-authors (I've also ensured their CLAs are signed) ?

**Comment by abhi-g** [comment]
Yes, its likely if there are commits in the PR with other authors

**Comment by MaanavD** [comment]
Alright, I've ensured that all e-mails used in the commit have signed the CLA. Hopefully the bot recognizes it. Hope to see this PR merged!

**Comment by sarahmaddox** [comment]
Preview:
https://deploy-preview-780--competent-brattain-de2d6d.netlify.com/docs/azure/

File: content/docs/azure/deploy/install-kubeflow.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+        wget <link to your release>
+
+1. Unpack the tar ball
+
+        tar -xvf kfctl_<release tag>_<platform>.tar.gz
+
+1. Run the following commands to set up and deploy Kubeflow. The code below includes an optional command to add the binary kfctl to your path. If you don’t add the binary to your path, you must use the full path to the `kfctl` binary each time you run it.
+```
+# The following command is optional, to make kfctl binary easier to use.
```
Comment (sarahmaddox) [inline content/docs/azure/deploy/install-kubeflow.md]: The formatting is broken from this point onwards. See the preview:
https://deploy-preview-780--competent-brattain-de2d6d.netlify.com/docs/azure/deploy/install-kubeflow/#the-following-command-is-optional-to-make-kfctl-binary-easier-to-use

To fix, try adding an empty line and indenting the code snippet. See this file for an example:
https://raw.githubusercontent.com/kubeflow/website/master/content/docs/gke/deploy/deploy-cli.md

File: content/docs/azure/deploy/uninstall-kubeflow.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
++++
+title = "Uninstall Kubeflow"
+description = "Instructions for uninstalling Kubeflow"
+weight = 10
++++
+
+## Uninstall Kubeflow on your Azure AKS cluster.
```
Comment (sarahmaddox) [inline content/docs/azure/deploy/uninstall-kubeflow.md]: Recommendation: change this from a heading to straight text. The page doesn't need a heading, as it's so short and the heading is equivalent to the page title.

Comment (MaanavD) [reply content/docs/azure/deploy/uninstall-kubeflow.md]: Apologies! I'll make some changes and send a PR right away.

**Comment by sarahmaddox** [comment]
Addresses https://github.com/kubeflow/website/issues/32

**Comment by sarahmaddox** [comment]
@MaanavD For the CLA, there's one commit with an email address that the CLA process can't identify. Please would you check? Sorry for all the admin on this. Once done, things should go smoothly. If that doesn't work then you may need to rebase the commits with the correct email address.

**Comment by MaanavD** [comment]
> @MaanavD For the CLA, there's one commit with an email address that the CLA process can't identify. Please would you check? Sorry for all the admin on this. Once done, things should go smoothly. If that doesn't work then you may need to rebase the commits with the correct email address.

Hi @sarahmaddox , I double and triple checked that myself as well as my two teammates have signed the CLA for all accounts we could have potentially used. Is there any way to figure out which e-mail we're having issues with?

**Comment by sarahmaddox** [comment]
@googlebot please re-check the CLA.

**Comment by sarahmaddox** [comment]
Thanks for confirming the CLA signatures @MaanavD

**Pull request merged**